scripts/pin_the_slow_way.py

```diff
@@ -63,6 +63,7 @@
           'libgdal': 'libgdal 2.1.*',  # 2.1.0
           'libmatio': 'libmatio 1.5.*',  # NA
           'libnetcdf': 'libnetcdf 4.4.*',  # 4.4.1
+          'libpcap': 'libpcap 1.*',  # NA
```

Comment:
This feels a little loose compared to the ABI report. Any thoughts on making it `1.8.*`?

Reply:
Yes, I felt so too. I'll make it `1.8.*`.

Reply:
Updated.